nav2_bringup/bringup/launch/bringup_launch.py

```diff
@@ -62,6 +65,21 @@ def generate_launch_description():
         'map',
         description='Full path to map yaml file to load')
 
+    declare_costmap_filters_cmd = DeclareLaunchArgument(
```

Comment:
Delete all the changes  to the launch files and parameter files. This should not be enabled by default, there is no keepout mask for the test world right now needing this.

Reply:
Removed all changes related to costmap filters in last update